_security-analytics/api-tools/correlation-eng.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: default
+title: Correlation engine APIs
+parent: API tools
+nav_order: 55
+---
+
+
+# Correlation engine APIs
+
+Correlation engine APIs allow you to create new correlation rules, view findings and correlations within a certain span of time, and perform other tasks.
+
+
+## Create correlation rules between log types
+
+Used to create correlation rules.
+
+```json
+POST /_plugins/_security_analytics/correlation/rules
+```
+
+### Request fields
+
+| Field | Type | Description |
+| :--- | :--- |:--- |
+| `index` | String | The name of the index used as the log source. |
+| `query` | String | The query used to filter security logs for correlation. |
+| `category` | String | The log type associated with the log source. |
+
+#### Example request
+
+```json
+POST /_plugins/_security_analytics/correlation/rules
+{
+  "correlate": [
+    {
+      "index": "vpc_flow",
+      "query": "dstaddr:4.5.6.7 or dstaddr:4.5.6.6",
+      "category": "network"
+    },
+    {
+      "index": "windows",
+      "query": "winlog.event_data.SubjectDomainName:NTAUTHORI*",
+      "category": "windows"
+    },
+    {
+      "index": "ad_logs",
+      "query": "ResultType:50126",
+      "category": "ad_ldap"
+    },
+    {
+      "index": "app_logs",
+      "query": "endpoint:/customer_records.txt",
+      "category": "others_application"
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "_id": "DxKEUIkBpIjg64IK4nXg",
+  "_version": 1,
+  "rule": {
+    "name": null,
+    "correlate": [
+      {
+        "index": "vpc_flow",
+        "query": "dstaddr:4.5.6.7 or dstaddr:4.5.6.6",
+        "category": "network"
+      },
+      {
+        "index": "windows",
+        "query": "winlog.event_data.SubjectDomainName:NTAUTHORI*",
+        "category": "windows"
+      },
+      {
+        "index": "ad_logs",
+        "query": "ResultType:50126",
+        "category": "ad_ldap"
+      },
+      {
+        "index": "app_logs",
+        "query": "endpoint:/customer_records.txt",
+        "category": "others_application"
+      }
+    ]
+  }
+}
+```
+
+### Response fields
+
+| Field | Type | Description |
+| :--- | :--- |:--- |
+| `_id` | String | The new rule's ID. |
+
+
+## List all findings and their correlations within a time window
+
+Provides a list of all findings and their correlations within a specified window of time.
```

Comment:
Please rephrase as a complete sentence naming the noun (what is used to do this?).